requirements.txt

```diff
@@ -1,6 +1,7 @@
 beautifulsoup4~=4.7
 cld2_cffi~=0.1
-datasketch==1.4.0
-spacy~=2.0
+datasketch==1.4
+spacy~=2.1
```

Comment:
```suggestion
spacy~=2.1.0
```

Reply:
Probably safer indeed. I'll revert this.